CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## [0.18.0](https://github.com/redhat-developer/vscode-quarkus/milestone/20?closed=1) (August 26, 2024)
```

Comment:
@angelozerr 0 -> 1 .

Reply:
It is too bad -( I cannoyt fix that since I created the release -( 

Good catch @rgrunber !